### ISSUE TITLE
Java client does not support openjdk

### ISSUE THREAD

**Issue by zxkane** [appium/appium#5551]
In fix #4763 introduced to check output of `java -version` with reg string **java version**, however it makes the java driver does not work with openjdk which is installed from ppa in Ubuntu 14.04.

```shell
/usr/bin/java -version
openjdk version "1.8.0_45-internal"
OpenJDK Runtime Environment (build 1.8.0_45-internal-b14)
OpenJDK 64-Bit Server VM (build 25.45-b02, mixed mode)
```

**Comment by davecroman** [comment]
+1. We need to update regex to support openjdk output.

https://github.com/appium/appium/blob/202c6e65ceb6a83d2573582cc500be4c868b52be/lib/devices/android/android-common.js

**Comment by rnovacek** [comment]
This seems to fix this issue for me:

```diff
-if (new RegExp("java version").test(line)) {
+if (new RegExp("(java|openjdk) version").test(line)) {
```

**Comment by jstephenson** [comment]
As I've run across this I may as well add that this fix appears to be in `master`, but notably not in `1.4.16` (stable). I assume therefore this will be rectified in `1.5.0`?

**Comment by scottdixon** [comment]
@jstephenson @zxkane - can anyone confirm 1.5 resolves this?

**Comment by jstephenson** [comment]
@scottdixon Yes, can confirm fixed in 1.5.

**Comment by scottdixon** [comment]
Great. Thanks for the update.
